desc/objectives/objective_funs.py

```diff
@@ -980,6 +1012,54 @@ def jac_unscaled(self, *args, **kwargs):
         """Compute Jacobian matrix of vector form of the objective wrt x, unweighted."""
         return self._jac_unscaled(*args, **kwargs)
 
+    def jvp_scaled(self, v, x, constants=None):
+        """Compute Jacobian-vector product of the objective function.
+
```

Comment:
Not necessary but it would be super helpful to add the matrix product operation in the docstring. Something like (A B) C^T or A (B C^T) and add a few words about why one is faster etc.